extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -1338,6 +1338,15 @@ public static enum ApplicationType {
     @ConfigItem
     public Optional<Provider> provider = Optional.empty();
 
+    /**
+     * Indirectly verify the user authentication has been successful with request to `user-info-path`.
+     * Bearer token is considered valid if `user-info-path` endpoint responded with `200 OK`.
+     * You should only enable this option with OIDC non-compliant provider without introspection endpoint.
+     * Please make sure 'user-info' endpoint is secured.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean allowBearerTokenVerificationWithUserInfo;
```

Comment:
Sorry, I've realized we can also have a case of the opaque access token being returned with the code flow, and the verify access token property (in `Authentication` group) is enabled. So I think we need to replace `BearerToken`  with `AccessToken` and add an `opaque` to make it crystal clear, so I propose: `verifyOpaqueAccessTokenWithUserInfo` as `allowOpaqueAccessTokenVerificationWithUserInfo` is a way too long :-). I'll update the previous suggestions accordingly